bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/AsterismTypeConverters.scala

```diff
@@ -29,8 +29,5 @@ object AsterismTypeConverters {
       (GhostHigh, GhostHighResolutionTargetPlusSky, GhostStandard) -> GhostTargetPlusSky,
       (GhostHigh, GhostHighResolutionTargetPlusSky, GhostPRV) -> GhostHighResolutionTarget,
 
-      (GhostPRV, GhostHighResolutionTarget, GhostStandard) -> GhostSingleTarget,
-
-      // This oddly happens sometimes?
-      (GhostStandard, GhostHighResolutionTarget, GhostHigh) -> GhostHighResolutionTarget)
+      (GhostPRV, GhostHighResolutionTarget, GhostStandard) -> GhostSingleTarget)
```

Comment:
Is the map complete? What would happen if we try to look up a conversion that isn't defined?

Reply:
Perhaps it should be something like:

```scala
val asterismTypeConverters: ((ResolutionMode, AsterismType, ResolutionMode)) => Option[Asterism Type] =
  Map(
    (GhostHigh, GhostHighResolutionTargetPlusSky, GhostStandard) -> GhostTargetPlusSky,
    ...
  ).lift
```
so that it is obvious some mappings may not be defined?